addons/l10n_cl/data/template/account.tax-cl.csv

```diff
@@ -1,101 +1,101 @@
-"id","name","description","invoice_label","amount","amount_type","type_tax_use","l10n_cl_sii_code","tax_group_id","active","sequence","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent","name@es"
-"ITAX_19","VAT 19% Sale","","VAT 19% Sale","19.0","percent","sale","14","tax_group_iva_19","","","base","invoice","+Ventas Netas Gravadas con IVA","","","IVA 19% Venta"
+"id","name","description","invoice_label","amount","amount_type","type_tax_use","l10n_cl_sii_code","tax_group_id","active","sequence","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","repartition_line_ids/factor_percent","description@es"
+"ITAX_19","19%","VAT 19% Sale","VAT 19% Sale","19.0","percent","sale","14","tax_group_iva_19","","","base","invoice","+Ventas Netas Gravadas con IVA","","","IVA 19% Venta"
 "","","","","","","","","","","","tax","invoice","+IVA Debito Fiscal","account_210710","",""
 "","","","","","","","","","","","base","refund","-Ventas Netas Gravadas con IVA","","",""
 "","","","","","","","","","","","tax","refund","-IVA Debito Fiscal","account_210710","",""
-"OTAX_19","VAT 19% Purchase","","VAT 19% Purchase","19.0","percent","purchase","14","tax_group_iva_19","","","base","invoice","+Compras Netas Gravadas Con IVA (recuperable)","","","IVA 19% Compra"
+"OTAX_19","19%","VAT 19% Purchase","VAT 19% Purchase","19.0","percent","purchase","14","tax_group_iva_19","","","base","invoice","+Compras Netas Gravadas Con IVA (recuperable)","","","IVA 19% Compra"
 "","","","","","","","","","","","tax","invoice","+IVA Pagado Compras Recuperables","account_110710","",""
 "","","","","","","","","","","","base","refund","-Compras Netas Gravadas Con IVA (recuperable)","","",""
 "","","","","","","","","","","","tax","refund","-IVA Pagado Compras Recuperables","account_110710","",""
-"I_IU2C","Ret. 2da Category 2020","","Ret. 2da 2020","-10.75","percent","purchase","15","tax_group_2da_categ","False","2","base","invoice","+Base Retención Segunda Categoría","","","Ret. 2da Categoría 2020"
+"I_IU2C","10.75% WH","Ret. 2da Category 2020","Ret. 2da 2020","-10.75","percent","purchase","15","tax_group_2da_categ","False","2","base","invoice","+Base Retención Segunda Categoría","","","Ret. 2da Categoría 2020"

```

Comment:
I'm not a Spanish expet, but shouldn't it be 

```suggestion
"I_IU2C","10.75% WH","Withholding 2nd Category 2020","Ret. 2da 2020","-10.75","percent","purchase","15","tax_group_2da_categ","False","2","base","invoice","+Base Retención Segunda Categoría","","","Ret. 2da Categoría 2020"

```
?

(also on subsequent lines)